examples/barycenters/plot_free_support_sinkhorn_barycenter.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
I agree with you! I added a section to the plot_free_support_barycenter.py so that readers may compare both. Nonetheless,  I noticed that even after tweaking the parameters of the Sinkhorn barycenter, the barycentric distribution is a bit strange. Could you take a look on the script? (last commit).

I also changed the distributions in the Sinkhorn barycenter example I added before, so that we do not test with rotations anymore. If you still think 2 examples on the matter is not necessary, I make the appropriate changes in the next commit.

Thanks for the help!